src/Algebra/Graph/Export/Dot.hs

```diff
@@ -71,8 +71,8 @@ defaultStyle v = Style mempty mempty [] [] [] v (\_ -> []) (\_ _ -> [])
 -- @
 -- defaultStyleViaShow = 'defaultStyle' ('fromString' . 'show')
 -- @
-defaultStyleViaShow :: (Show a, IsString s, Monoid s) => Style a s
-defaultStyleViaShow = defaultStyle (fromString . show)
+defaultStyleViaShow :: (Show a, IsString s, Escape s, Monoid s) => Style a s
+defaultStyleViaShow = defaultStyle (fromString . escape . show)
```

Comment:
Ahh, I just realised that since we can escape **after** converting to `String` we don't need the constraint, and we might not even need the typeclass!

Reply:
I've just realised this too... at least in the context of defaultStyleViaShow.  I don't know whether you'd agree Alga should sanitize some of the other paths e.g. exportAsIs and whether we'd need the typeclass for that.